### PR TITLE
Raise MissingDataError in query.py

### PR DESCRIPTION
**Pull request by Pokinawa**

### CODE REVIEW

File: adr/query.py [hunk 1]
```diff
@@ -51,6 +52,8 @@ def query_activedata(query):
                              data=query,
                              stream=True)
     response.raise_for_status()
+    if response.json().get('message') is None:
+        raise MissingDataError('ActiveData didn\'t return any data.')
```
Comment (worldomonation) [inline adr/query.py]: Issue with escaping single quotes can be bypassed if the string is formatted as:

`"ActiveData didn't return any data."`

Note the mixing of single and double quotes.

**Comment by Pokinawa** [comment]
This addresses the issue #41

File: adr/query.py [hunk 2]
```diff
@@ -51,6 +52,8 @@ def query_activedata(query):
                              data=query,
                              stream=True)
     response.raise_for_status()
+    if response.json().get('message') is None:
```
Comment (ahal) [inline adr/query.py]: The `message` key is specific to the `try_usage` query. So this should check if the `data` key itself is empty.

E.g:

    obj = response.json()
    if not obj.get('data'):
        ...
    return obj

**Comment by ahal** [comment]
Thanks for the PR! It needs a minor fix but is really close to being able to land.

**Comment by ahal** [comment]
Merged:
https://github.com/mozilla/active-data-recipes/commit/1b55bab79b30640aeb1a418fc030c1f85a9fdd31

**Pull request closed without merging**